documentation-website/Writerside/hi.tree

```diff
@@ -8,14 +8,15 @@
 
     <toc-element topic="Home.topic"/>
     <toc-element topic="Getting-Started-with-Exposed.md"/>
+    <toc-element topic="Exposed-Modules.md"/>
     <toc-element topic="Database-and-DataSource.md"/>
-    <toc-element topic="Transactions.md"/>
-    <toc-element topic="Deep-Dive-into-DSL.md"/>
-    <toc-element topic="Deep-Dive-into-DAO.md"/>
-    <toc-element topic="Exposed-Modules.md">
+    <toc-element topic="Table-Definition.md">
         <toc-element topic="Data-Types.md"/>
         <toc-element topic="SQL-Functions.md"/>
     </toc-element>
+    <toc-element topic="Transactions.md"/>
+    <toc-element topic="Deep-Dive-into-DSL.md"/>
+    <toc-element topic="Deep-Dive-into-DAO.md"/>
```

Comment:
@vnikolova This may not match exactly with the proposed structure on the drive doc, so let me know if I should change.

`Table-Definition.md` was not being used despite all its contents, so I've added it and used it as the parent of `Data-Types` and `SQL-Functions`, which makes more sense than the file with modules and dependencies.
I've also moved these to follow the content on databases, since schema definition seems to usually come after a connection is set up, but before a transaction is used.

Reply:
I'm not sure if `SQL-Functions` makes sense to be under `Table-Definitions` because it's not part of defining a Table, right? Perhaps we could have an empty `toc-element` such as "Working with Tables" that will be the parent of `Table-Definition`, `Data-Types` and `SQL-Functions`. What do you think?